src/Microsoft.AspNet.Mvc.ViewFeatures/IViewComponentHelper.cs

```diff
@@ -7,22 +7,27 @@
 
 namespace Microsoft.AspNet.Mvc
 {
+    /// <summary>
+    /// Supports the rendering of View Components in a view.
+    /// </summary>
     public interface IViewComponentHelper
     {
-        IHtmlContent Invoke(string name, params object[] args);
-
-        IHtmlContent Invoke(Type componentType, params object[] args);
-
-        void RenderInvoke(string name, params object[] args);
-
-        void RenderInvoke(Type componentType, params object[] args);
-
-        Task<IHtmlContent> InvokeAsync(string name, params object[] args);
-
-        Task<IHtmlContent> InvokeAsync(Type componentType, params object[] args);
-
-        Task RenderInvokeAsync(string name, params object[] args);
-
-        Task RenderInvokeAsync(Type componentType, params object[] args);
+        /// <summary>
+        /// Invokes a View Component with the specified <paramref name="name"/>.
```

Comment:
Is the casing for this `View Component` or `view component`? I saw some docs earlier which used the former and stuck with it.